docs/CW.md

```diff
@@ -146,17 +144,13 @@ A list of forms showing the derivational breakdown of the word, each separated b
 
 The date that the entry was last updated.
 
-## `\fststem` **FST stem**
-
-This field is _not_ in the Toolbox database. This data only lives in the TSV version of this file. There are somewhere in the range of ~1,500 entries total where this information has been added. This data should be retrieved and added to the main entry in the dictionary database. Forms marked as "CHECK" are ones that need FST stems.
```

Comment:
For FST stems marked with CHECK, we hadn't been able to resolve the FST stem. A typical reason is that the entries in question are derived or inflected forms of a lemma, and perhaps should not be considered as a regular lemma.